.github/workflows/deploy-docs.yml

```diff
@@ -63,23 +62,24 @@ jobs:
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          # Build directory contents to publish to `gh-pages` branch:
+          # Build directory contents to publish to the `gh-pages` branch:
           publish_dir: ./docs/site
           # Directory to place `publish_dir` contents on the `gh-pages` branch:
           destination_dir: ${{ env.DOCS_VERSION }}
           user_name: ${{ env.GIT_USER }}
           user_email: ${{ env.GIT_EMAIL }}
+
   add-version-to-docs:
+    name: 'Handle `versions.json` in the `gh-pages` branch'
```

Comment:
I think i would name it "add new version to selector":

![image](https://user-images.githubusercontent.com/20406381/110603674-8e209480-8187-11eb-8541-f84e7c1e6772.png)